streams/src/test/java/org/apache/kafka/streams/integration/EosBetaUpgradeIntegrationTest.java

```diff
@@ -837,6 +866,11 @@ public void init(final ProcessorContext context) {
                             crash = errorInjectedClient2;
                             sharedCommit = commitCounterClient2;
                         }
+                        punctuator = context.schedule(
+                            Duration.ofSeconds(5),
```

Comment:
Would it speed up the test to choose a smaller number here?

Reply:
I suppose it would. Does 100ms seem reasonable?

Reply:
Yeah, that's what I was thinking